### ISSUE TITLE
`storeutl` app ignores `-certs`, `-keys`, and `-crls` option

### ISSUE THREAD

**Issue by DDvO** [openssl/openssl#20144]
When preparing #20143, I noticed that `openssl storeutl` ignores any provided `-certs`, `-keys`, or `-crls` option.

For instance, this still shows any included keys and certs:
```
apps/openssl storeutl -passin pass:123456 test_123456.pem -crls

0: Pkey
-----BEGIN PRIVATE KEY-----
...
-----END PRIVATE KEY-----
1: Certificate
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
2: Certificate
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
3: Certificate
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
Total found: 4
```

**Comment by t8m** [comment]
Hmm... isn't this just a matter of putting -crls before the test_123456.pem filename?

**Comment by DDvO** [comment]
> Hmm... isn't this just a matter of putting -crls before the test_123456.pem filename?

Indeed then it works!

Yet there is no hint given for these options that they need to be given before the input file/URL,
and there is no warning or error if they are given thereafter.

Silently ignoring them is not good.

**Comment by DDvO** [comment]
I found that this is due to a general weakness of the `opt_check_rest_arg()` function, fixed by #20156.
